packages/graphql/src/constants.ts

```diff
@@ -56,10 +56,16 @@ export const RESERVED_TYPE_NAMES = [
     },
 ];
 
+export const RELATIONSHIP_TYPE_FIELD = "_type";
```

Comment:
A decision on what the field name should be for the `edge` type can be made on this line. The implementation makes use of this variable and the tests can be updated with a simple find and replace.